src/video.ts

```diff
@@ -0,0 +1,56 @@
+import {TranslationProject} from './translation_project'
+
+export class Videoplayer {
+    private videopanel: any;
+    private getExtensionFromPath(path: string){
+        return path.split('.').pop();
+    }
+    constructor(proj: TranslationProject) {
```

Comment:
I would prefer that the constructor accepted just mediaPath: string, but I'm ok with this.

Reply:
Corrected on commit f70b028